webpack/assets/javascripts/react_app/components/notifications/NotificationDropdown.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Dropdown, Glyphicon, MenuItem } from 'react-bootstrap';
+
+const NotificationDropdown = ({ links, id }) => {
+  const listLinks = links.map((link, i) => {
+    const liKey = `notification-link-${i}-${id}`;
+
+    return (
+      <MenuItem key={liKey} id={liKey} target="_blank" href={link.href}>
+        {link.title}
+      </MenuItem>
+    );
+  });
+
+  return (
+    <Dropdown className="pull-right dropdown-kebab-pf" pullRight key={id}
```

Comment:
@dLobatog you might find this useful [A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)